pkgs/by-name/lo/localai/package.nix

```diff
@@ -0,0 +1,245 @@
+{
+  # tinydream need to have at least gcc13
+  gcc13Stdenv
+, lib
+, fetchpatch
+, fetchFromGitHub
+, ncurses
+, abseil-cpp
+, protobuf
+, grpc
+, openssl
+, openblas
+  # needed for audio-to-text
+, ffmpeg
+, opencv
+, ncnn
+, sonic
+, spdlog
+, fmt
+, onnxruntime
+, cmake
+, buildGoModule
+, pkg-config
+, cudaPackages
+, makeWrapper
+, runCommand
+, buildType ? ""
```

Comment:
`buildType` deserves a comment about its possible values

Reply:
Possibles values are defined here:
https://github.com/mudler/LocalAI/blob/master/docs/content/docs/getting-started/build.md?plain=1#L86

this derivation handles following buildTypes: "" "cublas" "openblas"
Not supported is: "hipblas" "clblas" "szcl_f16" "szcl_f32" "metal"